### DISCUSSION TITLE
I would like to generate images on my laptop but it does not have a GPU. Is it still possible?

### DISCUSSION THREAD

**Discussion by ZiplEix** [lllyasviel/Fooocus#1155] [Q&A] [answered]
I have installed fooocus on my laptop but it does not have a GPU. When I try to generate an image I got an error that the image could not be generated because I do not have an NVIDIA driver on my computer (normal because I do not have a GPU). Is it still possible to generate an image?

**Comment by mel870** [comment, marked as answer]
Options are, using the Colab instructions, or if you need more GPU power,  you can rent a GPU and use the Linux install instructions. Here are some services that are fairly cheap.

1. https://runpod.io 
2. https://Vast.ai
3. https://Tensordock.com
4. https://paperspace.com

If you're not comfortable with setting up Fooocus on a Linux server, there are services that have it already setup. But they are more costly per hour than renting a GPU. 

1. https://rundiffusion.com/
2. https://www.thinkdiffusion.com/

**Reply by ZiplEix** [reply]
Thx
